File: erettsegi_emelt/2015_may_eng/c#/Tanc.cs
public class Tanc {

    public readonly string category;
    public readonly string woman;
    public readonly string man;

    public Tanc(string[] lines, int index) {
        this.category = lines[index];
        this.woman = lines[index + 1];
        this.man = lines[index + 2];
    }
}

File: erettsegi_emelt/2015_may_eng/java/Tanc.java
import java.util.*;

public final class Tanc {

    public final String category;
    public final String woman;
    public final String man;

    public Tanc(List<String> lines, int index) {
        this.category = lines.get(index);
        this.woman = lines.get(index + 1);
        this.man = lines.get(index + 2);
    }
}